src/display-buffer.coffee

```diff
@@ -465,6 +465,10 @@ class DisplayBuffer extends Model
 
     folds
 
+  # Returns all the folds that contain the given row range.
+  foldsContainingBufferRowRange: (startRow, endRow) ->
```

Comment:
A cursory search showed up that there's some reference to `foldsContainingBufferRow` in packages (some typescript header files), so I think it's just better to rename the new one only (7d97241). 

Thanks! :sparkles: